cppcheck-studio.py

```
@@ -0,0 +1,259 @@
+"""
+CPPCheck Studio - Unified CLI for C++ Static Analysis Dashboard Generation
+
+Transform CPPCheck JSON output into beautiful, interactive HTML dashboards.
+"""
+
+import argparse
+import json
+import sys
+import time
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)